packages/trader/src/Constants/__tests__/contract.spec.tsx

```diff
@@ -0,0 +1,370 @@
+import React from 'react';
+import { localize, Localize } from '@deriv/translations';
+import {
+    getCardLabels,
+    getMarketNamesMap,
+    getUnsupportedContracts,
+    getSupportedContracts,
+    getContractConfig,
+    getContractTypeDisplay,
+    getContractTypePosition,
+} from '../contract';
+
+const card_labels = {
```

Comment:
As the purpose of half of the functions is just to return specific data, I had to hardcode it in order to check whether it is equal to returning value (lines from 13 to 297)

Reply:
@kate-deriv Better to rethink what should be tested 🙏🏻 Now maintaining this test file is more challenging than the helper functions themselves 👀 

There is no need to write tests for `getCardLabels`, `getMarketNamesMap`, `getUnsupportedContracts`, `getSupportedContracts`, and `getContractConfig` as they don't contain any logic.

`getContractTypeDisplay` can be tested with different cases to ensure its logic will always return the correct value as expected.

Reply:
But `getContractTypeDisplay` call `getContractConfig`, `getSupportedContracts` and `getUnsupportedContracts` inside, so anyway you need all that data

Reply:
I cleared most of the hardcoded data👌🏻